designsystem/field/field-observer.ts

```diff
@@ -43,7 +43,9 @@ function handleMutation(fields: HTMLCollectionOf<Element>) {
 			renderDatalist(input, datalist);
 			renderCounter(input);
 			renderTextareaSize(input);
-			attr(input, "aria-describedby", descs.join(" "));
+			if(descs.length > 0) {
+        attr(input, "aria-describedby", descs.join(" "));
+      }
```

Comment:
Nice!! Kan vi gjøre sånn her? ☺️ Så fjerner den også attributen dersom den har hatt description, og denne fjernes
```suggestion
        attr(input, "aria-describedby", descs.join(" ") || null);
```